typescript/src/schema/proto.ts

```diff
@@ -35,6 +35,12 @@ const COMMON_PROTO_LIST = [
   'google.type',
 ];
 
+// services that are allowed to use Int32Value and UInt32Value protobuf wrapper types
+// instead of "number" for pageSize/maxResults
+// keyed by proto package name, e.g. "google.cloud.foo.v1".
+const ENABLE_WRAPPER_TYPES_FOR_PAGE_SIZE = {
```

Comment:
nit, this could be simplified if we just assume this list is opt-in, and remove the `true`flag, but up to you!